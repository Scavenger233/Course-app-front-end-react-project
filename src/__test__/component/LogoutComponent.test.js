import React, { Component } from 'react';

import LogoutComponent from '../.././component/LogoutComponent';
import renderer from 'react-test-renderer'

test("renders correctly", async() => {

    const tree = renderer.create(<LogoutComponent />).toJSON();
    expect(tree).toMatchSnapshot();
});
