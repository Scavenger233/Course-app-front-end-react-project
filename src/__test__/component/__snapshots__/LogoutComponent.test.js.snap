// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Array [
  <h1>
    You are logged out
  </h1>,
  <div
    className="container"
  >
    Thank You for Using Our Application.
  </div>,
]
`;
